usaspending_api/common/query_with_filters.py

```diff
@@ -132,7 +132,7 @@ def _generate_elasticsearch_query(cls, filter_values: List[str], query_type) ->
         fields = ["recipient_name"]
 
         for v in filter_values:
-            upper_recipient_string = v.upper()
+            upper_recipient_string = v.upper() + "*"
```

Comment:
What is the expected functionality? Currently in Prod if you search for "Boo" you only get two hits, but if you search for "Booz" you will get hits on Booz Allen Hamilton. Won't adding a wildcard make it so that "Boo" will also search for "Booz Allen Hamilton"?

Reply:
Yeah, that's the idea. This was discussed with the PO - we want searches to return results if they partially match the search text - i.e., `boo` to `Booz Allen` and `lab` to `labs`, `laboratory`, etc.

Reply:
Few small things:

- Should we be able to search for "ooz" as well? Would need a wildcard at the front of the string.
- The wildcard needs to be added in the conditional (or a different variable needs to be created) since this will have unexpected results with the `if` of the conditional below. 
- The AC for your story and mine (spending_over_time) needs to be udpated to reflect this decision since they currently say they should mimic current functionality.

Reply:
1. The PO did not specify, I've asked for follow up, but generally we want to avoid having leading wildcards, because they cause performance hits compared to trailing wildcards
2. Already dealt with
3. Sure

Reply:
Leading wildcards are discouraged due to performance concerns. I doubt there is a strong-enough desire for that functionality to incur the performance penalty